verilog/tools/kythe/indexing_facts_tree_extractor.cc

```diff
@@ -926,6 +932,12 @@ void IndexingFactsTreeExtractor::ExtractMethodCallExtension(
   // Move the data from the last sibling to the current node and delete that
   // sibling
   MoveAndDeleteLastSibling(function_node);
+  
+  // Terminate if no function name is found.
+  // in case of built-in functions: "sin()";
```

Comment:
change example from "sin" to "q.sort()" or "q.find()" (these are built-in array methods)